### PR TITLE
[ADD] Includes a way to classify claims adding the concept of type an…

### PR DESCRIPTION
**Pull request by yaniaular**
…d with this, it allows to configure claim stages depending on claim types. This module includes Customer, Supplier and Other claim types as default data and its own stages relation, and also makes few stages common for different claim types

### CODE REVIEW

**Comment by pedrobaeza** [comment]
For me it seems now correct (although there are some nitpicking like the title style or the version numbering whose conventions have changed during the progress of this PR), but I don't want to block it more.

:+1:

File: crm_claim_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```
Comment (lmignon) [inline crm_claim_type/README.rst:3]: Make the module title a real title by adding '======' also above the title.

**Comment by lmignon** [comment]
:+1: (Code review + functional tests)

**Comment by nhomar** [comment]
two +1 then I merge.

**Pull request merged**